### PR TITLE
Edit Profile pages

### PR DESCRIPTION
**Pull request by ashleynguyen27**
### What does this PR do?
Edit pages for the organization and volunteer profiles.

### Status
IN DEVELOPMENT

### Related PRs
List related PRs against other branches:

### Todos
- [ ] Testing

### Screenshots
[insert image(s) here]

CC: @steveshi7

### CODE REVIEW

File: app/javascript/components/organization/EditProfileForm.jsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        console.log(error);
+      });
+  };
+
+  render() {
+    const { organization} = this.state;
+    return {
+      <div className="w-100 h-100 bg-white">
+        <div className="tl fl w-75 ml6 mr6 mt6 mb5 bg-white pa5">
+            <
```
Comment (SaladRaider) [inline app/javascript/components/organization/EditProfileForm.jsx]: Remove this

File: app/javascript/components/organization/EditProfileForm.jsx [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+import axios from "axios";
+import Dropdown from '../../utils/Dropdown';
+
+class EditProfileForm extends React.components {
+  constructor(props) {
+    super(props);
+    var name = props.name;
+    var website = props.organization.website;
+    var location = props.organization.lcoation;
+    var passwrod = "*****";
```
Comment (SaladRaider) [inline app/javascript/components/organization/EditProfileForm.jsx]: misspelling of password. if this was taken from the `EditProjectForm`, then this is my fuk up haha.

File: app/javascript/components/organization/EditProfileForm.jsx [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    var profileImage = <span>No Image</span>;
+    if (this.props.profileImage_url) {
+      profileImage = <img src={this.props.profileImage_url} />;
+    }
+    var mission = props.organization.description;
+    var contact_first_name = props.organization.contact_first_name;
+    var contact_last_name = props.organization.contact_last_name;
+    var contact_phone_number = props.organization.phone_number;
+    var contact_selected_file = props.organization.contact_selected_file;
+    this.state{
```
Comment (SaladRaider) [inline app/javascript/components/organization/EditProfileForm.jsx]: missing `=` sign?

File: app/javascript/components/organization/EditProfileForm.jsx [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+                <input
+                    className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                    type="password"
+                    onChange={this.handleChange("password_confirmation")}
+                />
+              </div>
+            </div>
+            <div className="dib w-100 ml3">
+              <h3 className="mt3">Add a logo</h3>
+              <input
```
Comment (SaladRaider) [inline app/javascript/components/organization/EditProfileForm.jsx]: Look at Fred's PR on how he does file uploads. It's really pretty and I think we should use that for all file uploads.

File: app/javascript/components/volunteer/EditProfileForm.jsx [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+      });
+
+  }
+
+  render() {
+    const { volunteer} = this.state;
+    return {
+      <div className="w-100 h-100 bg-white">
+        <div className="tl fl w-75 ml6 mr6 mt6 mb5 bg-white pa5">
+            <
```
Comment (SaladRaider) [inline app/javascript/components/volunteer/EditProfileForm.jsx]: Remove this

File: app/javascript/components/volunteer/EditProfileForm.jsx [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+              <input
+                className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                type="file"
+                onChange={this.handleFileChange}
+                value={this.state.profileImage}
+              />
+            </div>
+            <div className="dib w-100 ml3">
+              <h3 className="mt3">Add a resume</h3>
+              <input
```
Comment (SaladRaider) [inline app/javascript/components/volunteer/EditProfileForm.jsx]: Use fred's file upload here too

File: app/javascript/components/volunteer/EditProfileForm.jsx [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+            </div>
+
+
+
+            <div className="dib w-100 ml3">
+              <h3 className="mt3">Add a photo</h3>
+              <input
+                className="dib essay-box bg-light-gray mt1 w-100 pa3 input"
+                type="file"
+                onChange={this.handleFileChange}
```
Comment (SaladRaider) [inline app/javascript/components/volunteer/EditProfileForm.jsx]: Fred's file upload

**Comment by ashleynguyen27** [comment]
deleting branch because of name convention

**Pull request closed without merging**